docs/source/deployorderer/ordererplan.md

```diff
@@ -0,0 +1,90 @@
+
+If you’re creating an ordering service, you must create this genesis block by specifying the necessary parameters in `configtx.yaml` and using the `configtxgen` tool to create the block.
+
+If you are adding a node to the system channel, the best practice is to bootstrapped using the latest configuration block of the system channel. Similarly, an ordering node added to the consenter of an application channel will be boostrapped using the latest configuration block of that channel.
+
+Note that the `configtx.yaml` that is shipped with Fabric binaries is identical to the [sample `configtx.yaml` found here](https://github.com/hyperledger/fabric/blob/master/sampleconfig/configtx.yaml), and contains the same channel "profiles" that are used to specify particular desired policies and parameters (for example, it can be used to which ordering nodes that are consenters in the system channel will be used in an application channel). When creating a channel (whether for an orderer system channel or an application channel), you specify a particular profile by name in your channel creation command, and that profile, along with the other parameters specified in `configtx.yaml`, are used to build the configuration block.
+
+You will likely have to modify one of these profiles in order to create your system channel and to create your application channels (if nothing else, you are likely to have to modify the sample organization names). Note that to create a Raft ordering service, you will have to specify an `OrdererType` of `etcdraft`.
+
+Check out the [tutorial on creating a channel](./create_channel/create_channel.html#the-orderer-system-channel) for more information on how to create a genesis block.
+
+### Creating profiles for application channels
+
+Both the system and all application channels are built using a `configtx.yaml` file. Therefore, when editing your `configtx.yaml` to create the genesis block for your system channel, you can also add profiles for any application channels that will be created on this network. However, note that while you can define any set of consenters for each channel, **every consenter added to an application channel must first be a part of the system channel**. You cannot specify a consenter that is not a part of the system channel. Also, it is not possible to control the leader of the consenter set. Leaders are chosen by the `etcd` protocol used by the ordering nodes.
+
+## Sizing your ordering node resources
+
+Because ordering nodes do not host a state database or chaincode, an ordering node will typically only have a single container associated with it. Like the “peer container” associated with the peer, this container encapsulates the ordering processes that order and commit transactions for all channels on which the ordering node is a consenter (ordering nodes also validating actions in particular cases). The ordering node storage includes the blockchain for all of channels on which the node is a consenter.
```

Comment:
```suggestion
Because ordering nodes do not host a state database or chaincode, an ordering node will typically only have a single container associated with it. Like the “peer container” associated with the peer, this container encapsulates the ordering processes that order and commit transactions for all channels on which the ordering node is a consenter (ordering nodes also validate actions in particular cases). The ordering node storage includes the blockchain for all of channels on which the node is a consenter.
```